src/vs/platform/severityIcon/common/severityIcon.ts

```diff
@@ -5,8 +5,6 @@
 
 import { Codicon } from 'vs/base/common/codicons';
 import Severity from 'vs/base/common/severity';
-import { problemsErrorIconForeground, problemsInfoIconForeground, problemsWarningIconForeground } from 'vs/platform/theme/common/colorRegistry';
-import { registerThemingParticipant } from 'vs/platform/theme/common/themeService';
 
```

Comment:
the css import is missing

Reply:
Thank you @aeschli ! Since you pointed this out, I realized I also didn't import it in my previous PR and when I went to fix it I noticed you fixed it [here](https://github.com/microsoft/vscode/pull/166713/files). Sorry about that and thank you 😅

I tried to add the import for this PR with `import 'vs/css!./severityIcon';` but I get the following linter warning:

```
Imports violates 'vs/base/common/** or vs/base/parts/*/common/** or vs/platform/*/common/** or tas-client-umd or @microsoft/1ds-core-js or @microsoft/1ds-post-js or vs/nls' restrictions. See https://github.com/microsoft/vscode/wiki/Source-Code-Organization
```

Which makes me think I'm doing something wrong